### DISCUSSION TITLE
Criteria for Stable Channel

### DISCUSSION THREAD

**Discussion by olivierjh** [rancher/rke2#4171] [Q&A] [answered]
What are the criteria for releasing a version into the stable channel of RKE2?

My organization uses cloud services to stand up ephemeral cloud environments on a regular basis, and we need to upgrade our on-prem prod kubernetes environment from v1.24+rke2 to either v1.25 or v1.26 to stay within the support window of the cloud services we rely on (the main difference being v1.25 recently became stable, whereas v1.26 resides under the latest channel). Can you clarify the risks I'd be taking on in upgrading our production environment to the latest version of rke2 rather than the most recent stable branch?

**Comment by brandond** [comment, marked as answer]
For RKE2 we won't release new minors into the stable channel until they're supported by Rancher. New patches in an existing minor are generally released to stable after a couple days (not on a friday, etc).

**Reply by brandond** [reply]
> My organization uses cloud services to stand up ephemeral cloud environments on a regular basis

K3s is probably a better fit for this, and its channels also have the benefit of not being constrained by compatibility with Rancher.
